usb/usb_host_hid/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "1.0.0"
+version: "1.0.1~1"

```

Comment:
1.0.1 should be enough, right?

Reply:
I think so.
But I have asked you about the description, because I understood that I don't really know when and what better to use. 

Anyway, changed to 1.0.1 right now.

Reply:
No problem! Here are some details about versioning https://docs.espressif.com/projects/idf-component-manager/en/latest/reference/versioning.html#versioning-scheme

Reply:
Thanks, I have updated that and verifyed with esp-idf as well. 
New example works well with both versions: 1.0.0 and 1.0.1.